crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -262,6 +313,7 @@ struct Fulfillment {
     executed_amount: eth::U256,
     #[serde_as(as = "Option<serialize::U256>")]
     fee: Option<eth::U256>,
+    flashloan_lender: Option<FlashloanLender>,
```

Comment:
I believe the flashloan information conceptually belongs to the settlement instead of each order individually.